### PR TITLE
Update/iotex antenna

### PR DESCRIPTION
**Pull request by ccali11**
@shanejearley - Removed `iotex-antenna` npm package as well as `iopay.ts` composable. Interfacing with IOTX using MetaMask including `switchNetwork` functionality is currently enabled.

@shanejearley & @hawyar - Please note that I did not remove `iotex-antenna` from the `services/crawler` / `@casimir/crawler` package. Will defer to you guys there. If you'd like me to remove that in this PR, just let me know.

~~Note: Found a bug with what appears to be missing polyfills for the `crypto` package, which is a dependency within one of the `node_modules`. The fix can be found in `vite.config.ts` as well as installing `crypto-browserify` npm package.~~ (All lies. Removed that garbage.)

### CODE REVIEW

**Comment by hawyar** [comment]
@ccali11 please remove it from the crawler package as well.

**Comment by ccali11** [comment]
You got it, @hawyar 👍

**Comment by ccali11** [comment]
@hawyar - what would you like me to do with `services/crawler/src/providers/Iotex.ts`? Delete? Comment out?

**Comment by hawyar** [comment]
Feel free to remove the file

**Comment by ccali11** [comment]
And all traces of `iotex-antenna` are gone!

File: services/crawler/package.json [hunk 1]
```diff
@@ -11,8 +11,7 @@
         "@aws-sdk/client-athena": "^3.163.0",
         "@aws-sdk/client-s3": "^3.120.0",
         "@aws-sdk/lib-storage": "^3.121.0",
-        "@iotexproject/iotex-address-ts": "^1.0.4",
-        "iotex-antenna": "^0.31.3"
+        "@iotexproject/iotex-address-ts": "^1.0.4"
```
Comment (shanejearley) [inline services/crawler/package.json:14]: rm

File: apps/web/package.json [hunk 2]
```diff
@@ -15,8 +15,8 @@
     "buffer": "^6.0.3",
     "d3": "^7.8.1",
     "ethers": "^5.7.2",
-    "iotex-antenna": "^0.31.6",
     "util": "^0.12.5",
+    "vite-plugin-pages": "^0.28.0",
```
Comment (shanejearley) [inline apps/web/package.json:19]: rm

**Review by shanejearley — CHANGES_REQUESTED** [review]

**Review by shanejearley — APPROVED** [review]

**Pull request merged**